embulk-core/src/main/java/org/embulk/spi/Exec.java

```diff
@@ -41,8 +40,9 @@ public static Injector getInjector() {
     }
 
     @Deprecated
-    public static Timestamp getTransactionTime() {
-        return Timestamp.ofInstant(session().getTransactionTimeInstant());
+    @SuppressWarnings("deprecation")  // https://github.com/embulk/embulk/issues/1292
+    public static org.embulk.spi.time.Timestamp getTransactionTime() {
```

Comment:
Just out of curiosity, why we removed  `org.embulk.spi.time.Timestamp` from import section and use it in code?

Reply:
That's because the class `org.embulk.spi.time.Timestamp` is deprecated by itself. If an entire class is deprecated, `import ...;` is warned for deprecation, and `@SuppressWarnings("deprecation")` does not work for `import`.